CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 - **NEW**: new op: `SGN` (sign of argument)
 - **NEW**: new kria remote op: `KR.DUR`
 - **FIX**: dim M in edit mode when metro inactive
+- **NEW**: new pattern ops: P.SHUF, PN.SHUF, P.REV, PN.REV, P.ROT, PN.ROT
```

Comment:
nitpicking: markdown for op names: `P.SHUF`?

Reply:
Good catch, fixed this up throughout the changelog [2be4ea9](https://github.com/monome/teletype/pull/196/commits/2be4ea981613808eb1b49955026c5d115e20a292)